### PR TITLE
Add chrony conf class

### PR DESCRIPTION
**Pull request by carbonin**
Added a class for managing the chronyd configuration file.

Currently only includes methods to add servers and remove servers.

### CODE REVIEW

File: lib/linux_admin/chrony.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module LinuxAdmin
+  class Chrony
+    def initialize(conf = "/etc/chrony.conf")
+      raise MissingConfigurationFileError, "#{conf} does not exist" unless File.exist?(conf)
+      @conf = conf
+    end
+
+    def clear_servers
+      data = File.read(@conf)
+      data.gsub!(/server\s+.+\n/, "")
```
Comment (jrafanie) [inline lib/linux_admin/chrony.rb]: Is it possible to have comments or other things in this file with server in it?
Should we check that the beginning of the line starts with server?

File: lib/linux_admin/chrony.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      File.write(@conf, data)
+    end
+
+    def add_servers(*servers)
+      data = File.read(@conf)
+      data << "\n" unless data.end_with?("\n")
```
Comment (jrafanie) [inline lib/linux_admin/chrony.rb]: Can we instead do `data << "\nserver #{s}"` below?

Comment (carbonin) [inline lib/linux_admin/chrony.rb]: I couldn't decide what the best way to deal with this was.
I wanted to deal with the case where the file didn't end with a newline, but also make sure the file did end with a newline when I was finished with it.

Comment (jrafanie) [inline lib/linux_admin/chrony.rb]: Ok, I'm good with that.

**Comment by jrafanie** [comment]
@carbonin Can you coordinate a new release with this change so you can update your other PR to require that new version of linux_admin?

**Comment by carbonin** [comment]
@jrafanie Will do

**Pull request merged**